src/pudl/metadata/resources/eia861.py

```diff
@@ -388,6 +483,8 @@
         "etl_group": "eia861",
     },
     "core_eia861__yearly_net_metering_misc": {
+        # TODO: I feel skeptical that the pv_current_flow_type field shouldn't be linked to the other net metering table.
```

Comment:
Why did it get split out? Agree it feels a little funny to have a single data column over there all lonely.  If it were consolidated, would it be weird that the "non-net metering" data has a misc table, but the net metering doesn't?

Reply:
To be addressed in #3301